src/elements/common/sub-header/__tests__/SortButton.test.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { SortButtonBase } from '../SortButton';
```

Comment:
we could put the named import last after the default ones (specifically for this component section of imports)